backend/src/main/kotlin/app/ehrenamtskarte/backend/verification/service/CardActivator.kt

```diff
@@ -0,0 +1,21 @@
+package app.ehrenamtskarte.backend.verification.service
+
+import app.ehrenamtskarte.backend.verification.database.TOTP_SECRET_LENGTH
+import com.eatthepath.otp.TimeBasedOneTimePasswordGenerator
+import javax.crypto.KeyGenerator
+
+object CardActivator {
+
+    public fun generateTotpSecret(): ByteArray {
```

Comment:
Maybe try to wrap this into: https://kotlinlang.org/api/latest/jvm/stdlib/kotlin/synchronized.html